thruster-server/src/server.rs

```diff
@@ -14,93 +14,13 @@ use thruster_core::http::Http;
 use thruster_core::request::Request;
 use thruster_core::response::Response;
 
-// use std::thread;
-// use num_cpus;
-// use net2::TcpBuilder;
-// #[cfg(not(windows))]
-// use net2::unix::UnixTcpBuilderExt;
-
 use crate::thruster_server::ThrusterServer;
 
 pub struct Server<T: 'static + Context<Response = Response> + Send> {
     app: App<Request, T>,
 }
 
-// impl<T: 'static + Context<Response = Response> + Send> Server<T> {
```

Comment:
I have a todo to actually fix this code and uncomment it -- keep it around for the time being as (when it's running correctly,) has a large impact on benchmarks.